src/sentry/consumers/__init__.py

```diff
@@ -337,6 +337,10 @@ def build_consumer_config(group_id: str):
 
         if max_poll_interval_ms is not None:
             consumer_config["max.poll.interval.ms"] = max_poll_interval_ms
+            # HACK: If the max poll interval is less than 45 seconds, set the session timeout
+            # to the same. (it's default is 45 seconds and it must be >= to max.poll.interval.ms)
```

Comment:
typo: `max.poll.interval.ms` must be >= `session.timeout.ms`

Reply:
oops, thanks

Reply:
oops, thanks